libtiledbsoma/src/soma/soma_array.cc

```diff
@@ -327,7 +327,10 @@ void SOMAArray::write(bool sort_coords) {
     }
     mq_->submit_write(sort_coords);
 
-    mq_->reset();
+    // When we evolve the schema, the ArraySchema needs to be updated to the
```

Comment:
what happens if the user had opened the array for writing at a time less than the most recent write?  For example:
1. Create a ts=1, close
2. Open at ts=3, write, close
3. Open at ts=2, write, close

At step three, are things OK with the evolve?

Reply:
No, this is a known issue related to enumerations at timestamps: https://github.com/single-cell-data/TileDB-SOMA/issues/2879